.github/workflows/rust.yml

```diff
@@ -25,6 +25,16 @@ jobs:
       uses: arduino/setup-protoc@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
+    - if: matrix.os == 'ubuntu-latest'
+      name: Install dependencies (ubuntu-latest)
+      run: |
+        sudo apt update
+        sudo apt-get install clang libclang-dev llvm
+
+    - if: matrix.os == 'macOS-latest'
+      name: Dynamic Link libclang (macOS-latest)
+      run: echo "DYLD_FALLBACK_LIBRARY_PATH=$(xcode-select --print-path)/Toolchains/XcodeDefault.xctoolchain/usr/lib/" >> $GITHUB_ENV
```

Comment:
some weird macOS issue where libclang is not linked properly by default
found this trick on several Github repos

Reply:
LOL! This is in my `.profile`. 😂💀

<img width="720" alt="Screenshot 2025-01-10 at 12 04 58" src="https://github.com/user-attachments/assets/02ce4311-f7a8-4f66-b3cf-93b67eb1ab38" />